### PR TITLE
reduced font weight to 400 for label class

### LINKED ISSUE #5388: Reduce the font-weight of the contents in the tags buttons
Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

🤔 What you will need to know.
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

📋 Step by Step
 🙋 Claim this issue: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

📝 Update the file [app/assets/stylesheets/style.css](https://github.com/publiclab/plots2/blob/master/app/assets/stylesheets/style.css) in the plots2 repository.

 Currently the font-weight of the content in the tags button is 700. Reduce the font-weight to increase readability.
In style.css, add font-weight in the label class. 

For testing the changes made, it can be seen in the tags mentioned in the sidebar of any page or in the dashboard where the tags have been mentioned.

Screen Shot 2019-04-07 at 12 53 02 AM

 
💾 Commit your changes

🔀 Start a Pull Request. There are two ways how you can start a pull request:

If you are familiar with the terminal or would like to learn it, here is a great tutorial on how to send a pull request using the terminal.

You can also edit files directly in your browser and open a pull request from there.

🏁 Done Ask in comments for a review :)
Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update within a week of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

Is someone else already working on this?
🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- If someone seems stuck, offer them some help! Otherwise, take a look at some other issues you can help with. Thanks!

🤔❓ Questions?
Leave a comment below!

### PR DESCRIPTION
**Pull request by brjinks**
Fixes #5388  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by brjinks** [comment]
Can I get help from @publiclab/reviewers ? Thanks.

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -3,57 +3,65 @@ Junction is provided under the OFL and is available at http://www.theleagueofmov
...
-  font-family: 'Junction Regular';
-  src: url('/lib/junction/webfonts/junction-regular.eot');
-  src: url('/lib/junction/webfonts/junction-regular.eot?#iefix') format('embedded-opentype'),
-       url('/lib/junction/webfonts/junction-regular.woff') format('woff'),
-       url('/lib/junction/webfonts/junction-regular.ttf') format('truetype'),
-       url('/lib/junction/webfonts/junction-regular.svg#junctionregular') format('svg');
+  font-family: "Junction Regular";
+  src: url("/lib/junction/webfonts/junction-regular.eot");
+  src: url("/lib/junction/webfonts/junction-regular.eot?#iefix")
+      format("embedded-opentype"),
```
Comment (ananyaarun) [inline app/assets/stylesheets/style.css]: @brjinks , Can u pls remove the new line added here.

File: app/assets/stylesheets/style.css [hunk 2]
```diff
@@ -3,57 +3,65 @@ Junction is provided under the OFL and is available at http://www.theleagueofmov
...
+    url("/lib/junction/webfonts/junction-regular.woff") format("woff"),
+    url("/lib/junction/webfonts/junction-regular.ttf") format("truetype"),
+    url("/lib/junction/webfonts/junction-regular.svg#junctionregular")
+      format("svg");
```
Comment (ananyaarun) [inline app/assets/stylesheets/style.css]: The new line can be removed here as well

**Comment by ananyaarun** [comment]
@brjinks , i see u have made a lot of changes in terms of changing the indentation and spacing of the code in the css files. Kindly change only the font weight.
Also do add a screenshot of the change once done.
Great start, feel free to ask for help if stuck :) Thanks !!

**Comment by brjinks** [comment]
thanks @ananyaarun . I'm wondering if all those indentation changes were made automatically by the Prettier extension I had installed in my code editor. Should I make a new request then?

**Comment by brjinks** [comment]
I used git reset. When I tried to push the changes, I got an error because my branch is behind is remote counterpart. I tried following the Publiclab Git Workflow by pulling the latest version from the publiclab repo and running git rebase master. Now I keep getting "Merge conflict" messages and I don't know how to resolve them.

**Comment by gauravano** [comment]
Hi @brjinks, although there wasn't any need to delete the fork but I guess, it done and fine. For the future ref, you can see https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch for resetting the master branch and also feel free to ask any questions. Thanks!

I guess, you opened new PR at https://github.com/publiclab/plots2/pull/5535 so closing this one.

**Pull request closed without merging**